ge/org.osate.ge.errormodel/src/org/osate/ge/errormodel/ui/swt/TypeTokenListEditorDialog.java

```diff
@@ -148,15 +148,15 @@ protected Control createDialogArea(final Composite parent) {
 			this.addTypesButton.addSelectionListener(new SelectionAdapter() {
 				@Override
 				public void widgetSelected(final SelectionEvent e) {
-					final ErrorTypesSelectorModel selectorModel = new ErrorTypesSelectorModel(editorModel, (t) -> true,
+					final ErrorTypesSelectorModel selectorModel = new ErrorTypesSelectorModel(editorModel,
```

Comment:
Remove the parenthesis around (t). This will be considered a code smell in sonar cloud.